src/Network/Network/ChangeLog.md

```diff
@@ -38,6 +38,10 @@
 	- `Get-AzVpnClientConfiguration`
 	- `New-AzVirtualNetworkGateway`
 	- `Set-AzVirtualNetworkGateway`
+* Updated cmdlets to enable setting of PrivateRange on AzureFirewallPolicy.
+    - `New-AzFirewallPolicy`
+    - `Set-AzFirewallPolicy`
+
```

Comment:
Hi @kenchenMicrosoft , the log msgs should be added under the top of ## Upcoming Release section. Please merge network-feb into your branch, then you will detect that there should be a conflict.

Reply:
updated accordingly